lib/generator.rb

```diff
@@ -35,7 +37,22 @@ def sha1
   end
 
   def test_cases
-    cases.call(data)
+    cases.new(data).to_a
```

Comment:
Could we insert the bookkeeping test case into the cases array to always generate it at the end of the view?  This would allows us to render it just like all the other test cases, since it is a ExerciseTestCase